From 54fb45d0c744edda68a5f4f0ea31ba059023d973 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 4 Feb 2013 13:45:55 +0100
Subject: [PATCH] build: fix for automake-1.13

Use AC_CONFIG_HEADER instead of deprecated AM_CONFIG_HEADER.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e8a4ddd..2a5c72c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,7 +4,7 @@ AC_INIT([spice-gtk], [m4_esyscmd(build-aux/git-version-gen .tarball-version)],
         [spice-devel@lists.freedesktop.org])
 
 AC_CONFIG_MACRO_DIR([m4])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADER([config.h])
 AC_CONFIG_AUX_DIR([build-aux])
 
 AM_INIT_AUTOMAKE([foreign dist-bzip2 -Wall -Werror])
-- 
1.8.1.2

